src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -64,4 +64,5 @@ function GuiDrop(...)
 	let l:fnames = deepcopy(a:000)
 	let l:args = map(l:fnames, 'fnameescape(v:val)')
 	exec 'drop '.join(l:args, ' ')
+	doautocmd BufEnter
```

Comment:
FWIW, this is no longer required on master, it was a 30-year-old Vim bug that [was fixed](https://github.com/neovim/neovim/commit/42c922b32c0a22fbe078d03bac5de0cfa7bd0b9f).

It's harmless to leave it here, though it will cause a redundant BufEnter event now. Could wrap it in a `has('nvim-0.2')` check.